home/tests.py

```diff
@@ -1,3 +1,83 @@
+from .models import Question, Answer, Profile
+from django.test import Client
+from django.urls import reverse
+from datetime import datetime
+from django.db.models.query import QuerySet
+import pytest
 
-def test_first():
-    assert True
+
```

Comment:
Since you're going to end up with a very big tests file, perhaps you want to split it up ahead of time, or at least use classes to group related tests together.

Reply:
Classes were added to group-related tests.